utils/src/main/java/com/revenuecat/purchases/utils/Responses.kt

```diff
@@ -1,7 +1,9 @@
 package com.revenuecat.purchases.utils
 
+import org.json.JSONObject
+
 object Responses {
-    const val validFullPurchaserResponse = """
+    val validFullPurchaserResponse = JSONObject("""
```

Comment:
I was facing weird assertion errors when doing:
```
        assertThat(deserializedResult.httpResult.payload).isEqualTo(responsePayload)
 ```
 and realized that it was due to the formatting of the JSON. To remove any formatting I realized the best would be to create a JSONObject and then transform it to string again.